WebContent/js/containers/Filters.js

```diff
@@ -148,6 +163,8 @@ export class Filters extends React.Component {
         const { dispatch } = this.props;
         e.preventDefault();
         this.toggleFilters();
+        const { owner, prefix, status, jobId } = this.props;
```

Comment:
prop deconstruction should all happen in the same place

Reply:
moved, thanks!